lib/rspec/core/configuration.rb

```diff
@@ -226,7 +227,8 @@ def output_stream=(value)
             "`output_stream` will be ignored. You should configure it earlier for " \
```

Comment:
Mind updating the yard doc on this method to document it can be a string since it apparently can be?